README.md

```diff
@@ -12,52 +12,105 @@
+
+<!-- TODO: a section here that explains that you select plugins to configure behavior and how data goes through the system -->
+
+<!-- TODO: a cool diagram here that clearly demonstrates data moving through the system -->
+
+For example, use conduit to:
+* Build a notification system for on chain events.
+* Power a next generation block explorer.
+* Select app specific data and write it to a custom database.
+* Build a custom Indexer for a new [ARC](https://github.com/algorandfoundation/ARCs).
+* Send blockchain data to another streaming data platform for additional processing (e.g. RabbitMQ, Kafka, ZeroMQ).
+* Build an NFT catalog based on different standards.
 
 # Getting Started
 
-See the [Getting Started](./docs/GettingStarted.md) page.
+## Installation
+
+### Download
+
+The latest `conduit` binary can be downloaded from the [GitHub releases page](https://github.com/algorand/conduit/releases).
 
-## Building from source
+### Docker
 
-Development is done using the [Go Programming Language](https://golang.org/), the version is specified in the project's [go.mod](go.mod) file. This document assumes that you have a functioning
-environment setup. If you need assistance setting up an environment please visit
-the [official Go documentation website](https://golang.org/doc/).
+[The latest docker image is on docker hub.](https://hub.docker.com/r/algorand/conduit)
 
-Run `make` to build Conduit, the binary is located at `cmd/conduit/conduit`.
+### Install from Source
 
-# Configuration
+1. Checkout the repo, or download the source, `git clone https://github.com/algorand/conduit.git && cd conduit`
+2. Run `make conduit`.
+3. The binary is created at `cmd/conduit/conduit`.
 
-See the [Configuration](./docs/Configuration.md) page.
+## Usage
 
-# Third Party Plugins
+Conduit is configured with a YAML file named `conduit.yml`. This file defines the pipeline behavior by enabling and configuring different plugins.
 
-See the [Third Party Plugins](./docs/Third_Party_Plugins.md) page.
+### Create `conduit.yml` configuration file
 
-# Develoment
+Use the `conduit init` subcommand to create a configuration template. Place the configuration template in a new data directory. By convention the directory is named `data` and is referred to as the data directory.
 
-See the [Development](./docs/Development.md) page for building a plugin.
+```sh
+mkdir data
+./conduit init > data/conduit.yml
+```
 
-# Plugin System
 A Conduit pipeline is composed of 3 components, [Importers](./conduit/plugins/importers/), [Processors](./conduit/plugins/processors/), and [Exporters](./conduit/plugins/exporters/).
-Every pipeline must define exactly 1 Importer, exactly 1 Exporter, and can optionally define a series of 0 or more Processors.
+Every pipeline must define exactly 1 Importer, exactly 1 Exporter, and can optionally define a series of 0 or more Processors. A full list of available plugins with `conduit list` and the [plugin documentation page](TODO: plugin docs).
```

Comment:
This TODO is the last thing needed for the main README file. We could link to the [plugin package](https://github.com/algorand/conduit/tree/master/conduit/plugins) and expect people to click around, but that is a little awkward. Alternatively we can do a little scripting to copy README files into the docs directory (we already have some scripts that do this for the docs website).